src/stdatamodels/jwst/datamodels/wcs_ref_models.py

```diff
@@ -843,7 +960,7 @@ def __init__(self, init=None, apertures=None, **kwargs):
 
     @property
     def aperture_names(self):
-        return [getattr(ap, 'aperture_name') for ap in self.apertures]
+        return [getattr(ap, "aperture_name") for ap in self.apertures]  # noqa: B009
```

Comment:
why ignore instead of fixing?

Reply:
This line isn't covered by any tests. I chose not to change the code since I'm not if the change would be tested.

Reply:
should I open another issue requesting the addition of unit tests for `wcs_ref_models.py`?

Reply:
Thanks. Yeah, we might also consider a catch all issue that just says "add coverage for noqa lines". What do you think?

Reply:
I think that would be good, something like "Add unsafe style fixes, with test coverage"

Reply:
How does https://github.com/spacetelescope/stdatamodels/issues/390 look?